config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  devise_for :users, :controllers => {registrations: "registrations"}
+  resources :posts
```

Comment:
## [required] Routes
- [ ] The routes should create the application routes using resources :posts, only: [:new, :create, :index] ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) from Odin)